packages/iTwinUI-react/src/core/Footer/FooterSeparator.tsx

```diff
@@ -0,0 +1,31 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import { useTheme } from '../utils';
+import cx from 'classnames';
+import '@itwin/itwinui-css/css/footer.css';
+
+export type FooterSeparatorProps = Omit<
+  React.ComponentPropsWithRef<'li'>,
+  'children'
+>;
+
+/**
+ * Footer separator. Recommended to use with `Footer`.
+ */
+export const FooterSeparator = (props: FooterSeparatorProps) => {
+  const { className, ...rest } = props;
+  useTheme();
+
+  return (
+    <li
+      className={cx('iui-legal-footer-separator', className)}
+      role='separator'
```

Comment:
i think `aria-hidden` would be better as these are not real separators and there are two many of them

Reply:
Changed